pkg_resources/__init__.py

```diff
@@ -365,7 +365,7 @@ def get_provider(moduleOrReq):
     return _find_adapter(_provider_factories, loader)(module)
 
 
-def _macosx_vers(_cache=[]):
+def _macos_vers(_cache=[]):
```

Comment:
While we're at it, let's rename these to `_macOS_vers`. I prefer to honor the casing when possible.